dashboard/templates/dashboard/create.html

```diff
@@ -0,0 +1,90 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+{% if error %}
+{{ error }}
+<br />
+<br />
+{% endif %}
+
+<h1>Create</h1>
+
+<form method="POST" action="{% url 'create' %}">
+{% csrf_token %}
+<br />
+
+    <br />
+    <h3>Title:</h3>
+    <input type="text" name="title" />
+
+    <br />
+    <h3>URL:</h3>
+    <input type="text" name="url" />
+    <br />
+
+    
+    <div class="container", id="department">
+        <h3>Department:</h3>
+        <form>
+          <label class="checkbox-inline">
+            <input type="checkbox" value="">Software Engineering
+          </label>
+          <label class="checkbox-inline">
+            <input type="checkbox" value="">Product Management
+          </label>
+          <label class="checkbox-inline">
+            <input type="checkbox" value="">Interaction Design
+          </label>
+          <label class="checkbox-inline">
+            <input type="checkbox" value="">Science, Technology & Society
+          </label>
+          <label class="checkbox-inline">
+            <input type="checkbox" value="">Other
+          </label>
+        </form>
+    </div>
+
+    <div class="container", id="media_type">
+        <h3>Type:</h3>
+        <form>
+            <div class="form-group">
+                <label for="sel1">Select one:</label>
+                <select class="form-control" id="sel1">
+                    <option>Book</option>
+                    <option>Link</option>
+                    <option>Magazine</option>
+                    <option>Video</option>
+                    <option>Image</option>
+                    <option>Online Course</option>
+                    <option>Article</option>
+                    <option>Software/Tools</option>
+                    <option>Slides</option>
+                    <option>Audio/Podcast</option>
+                    <option>Library</option>
+                    <option>Database</option>
+                    <option>Blog</option>
+                    <option>Other</option>
+                </select>
+            </div>
```

Comment:
For this, I would loop over the info we have from the database!

Reply:
Great idea. I'll be editing it in a new branch so it's still rough, but I'll make sure to add that in.